lang/sv/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: swedish
+title: Semantisk versionshantering 2.0.0
+---
+
+Semantisk versionshantering 2.0.0
+==============================
+
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
```

Comment:
Excellent. That reads really well indeed.

Sent from my iPhone

> On 29 Jul 2016, at 12:36, Pär Björklund <notifications@github.com> wrote:
> 
> In lang/sv/index.md:
> 
> > @@ -0,0 +1,249 @@
> > +---
> > +layout: swedish
> > +title: Semantisk versionshantering 2.0.0
> > +---
> > +
> > +Semantisk versionshantering 2.0.0
> > +==============================
> > +
> > +Sammanfattning
> > +-------
> > +
> > +Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
> I would reword this whole section as
> 
> Ett versionsnummer består av tre delar MAJOR.MINOR.PATCH och det finns regler för hur de ska ändras.
> 1. MAJOR ökas med 1 när ändringarna påverkar bakåtkompatibilitet, t ex att ett API ändras eller en metod tas bort.
> 1. MINOR ökas med 1 när ny funktionalitet tillkommer men 100% bakåtkompatibilitet bibehålls, t ex nya metoder läggs till.
> 1. PATCH ökas med 1 vid buggfixar förutsatt att 100% bakåtkompatibilitet bibehålls.
> 
> Ibland kan det även förekomma ytterligare delar i versionsnumret för att markera t ex en test eller betaversion.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>